### ISSUE TITLE
Keystone C# binding 

### ISSUE THREAD

**Issue by aquynh** [ShaneK2/inVtero.net#3]
it is awesome to see the C# binding for Keystone+Capstone here! what do you think about putting that into separate independent repos, so they can be more accessible for others?

btw, there is one .NET binding for Keystone at https://github.com/chaplin89/Keystone.Net. what do you think about that?

keep it up, cheers.

**Comment by K2** [comment]
Yes I will clean up and drop the Capstone one into a new repro.  Thanks for the keystone link.  I had not seen that one when I started, I like how it has the symbol resolver ready, good to see.

**Comment by K2** [comment]
@aquynh 
I moved the Capstone.cs I was using into it's own repository [Capstone.NET](https://www.github.com/K2/Capstone.NET)

So this is a 64 bit version specifically, people can use the other one for 32 still or I'll eventually port it back, I also updated it with capstone 3.x and 4.x support.  I'll be adding a netcore/cross platform build as well shortly.

**Comment by aquynh** [comment]
wow thanks for the head up, keep up your awesome work!
-- 
Thanks,
Quynh

http://www.keystone-engine.org
http://www.capstone-engine.org
http://www.unicorn-engine.org

**Comment by K2** [comment]
Here's the Linux / Windows portable version for .NET x64 https://github.com/K2/CapstoneCore

I'll keep porting more arch as well.

**Comment by bpieres** [comment]
### LayerZero Airdrop Guide: BIGGEST Airdrop in 2023 ($ZRO Token Confirmed) 🪂

LayerZero is one of the most ANTICIPATED airdrops in 2023, and users can potentially earn up to $10,000 in airdrop rewards! This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Claim Now on Layerzero Oficial](http://link.layerzer0network.online/rFz335)

![Claim Now](https://i.ytimg.com/vi/s7BR96pg0SA/sddefault.jpg)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).
   - 
2. **Share on Social Media:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&text=LayerZero+Airdrop+Guide)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&text=LayerZero+Airdrop+Guide)
   - **Instagram:**
     [Share on Reddit](https://reddit.com/submit?url=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&title=LayerZero+Airdrop+Guide) 

2. **Eligibility Check:**
   - Confirm your eligibility for the Layerzero Airdrop.
